pegasus/test/test_form_helpers.rb

```diff
@@ -57,7 +58,7 @@ class FormHelpersTest < SequelTestCase
     it 'inserts a new form' do
       row = insert_or_upsert_form(
         'HocSignup2017',
-        DEFAULT_DATA.merge(email: "#{SecureRandom.hex(8)}@example.com")
+        DEFAULT_DATA.merge(email_s: "#{SecureRandom.hex(8)}@example.com")
```

Comment:
An unrelated bug fix surfaced by my refactoring the `insert_or_upsert_form` method.